### PR TITLE
added product-listing-page

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**
image

### CODE REVIEW

**Review by vivekgudipudi — APPROVED** [review]

**Comment by smkmallik** [comment]
Looking good, UI is lovely.

**Review by smkmallik — APPROVED** [review]
No issues on the code, seems good to merge

File: components/product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+.filter-wrapper {
+    display: flex;
+    gap: var(--s);
+    flex-direction: column;
+    min-width: 15rem;
+    position: relative;
+    padding-top: 3.75rem;
+    border-right: 2px solid var(--disabled-color);
+
```
Comment (samyakshah3008) [inline components/product-listing/product-listing.css:14]: Can remove this white space later on

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions, clean code and UI 
Writing meaningful comments made good readability for reviewing! 
Good for approval ✅

**Review by pavan6115 — APPROVED** [review]
UI looks clean ✅

**Pull request merged**